lib/wifi_iot.dart

```diff
@@ -98,12 +126,17 @@ class WiFiForIoTPlugin {
     return htResult;
   }
 
+  /// Set WiFi AP Configuaration
+  @Deprecated("This is will only work with < Android SDK 26.")
   static void setWiFiAPConfiguration(Object poWiFiConfig) async {
     Map<String, bool> htArguments = Map();
     htArguments["wifi_config"] = poWiFiConfig as bool;
     await _channel.invokeMethod('setWiFiAPConfiguration', htArguments);
   }
 
+  /// Get WiFi AP SSID
+  @Deprecated(
+      "This is will only work with < Android SDK 26. It could be make to work for >= Android SDK 29, request at https://github.com/alternadom/WiFiFlutter/issues/134.")
```

Comment:
minor typo in error message: `make` -> `made`

Reply:
Resolved in 7f5909a3b4dd935c522bd3a8888b7840f7ddeadf